src/driver.ml

```diff
@@ -1178,7 +1194,8 @@ let standalone_run_as_ppx_rewriter () =
     List.map standalone_args ~f:(fun (arg, spec, _doc) ->
       (arg, spec, " Unused with -as-ppx"))
   in
-  let args = List.rev_append !args standalone_args in
+  let args = get_args () ~standalone_args in
```

Comment:
(I would reverse the order of the arguments passed to `get_args`.)

Reply:
sure